go.mod

```diff
@@ -149,13 +149,15 @@ require (
 
 require (
 	github.com/Jille/raft-grpc-transport v1.2.0
+	github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d
 	github.com/arangodb/go-driver v1.3.2
 	github.com/fluent/fluent-logger-golang v1.9.0
 	github.com/hanwen/go-fuse/v2 v2.1.0
 	github.com/hashicorp/raft v1.3.9
 	github.com/hashicorp/raft-boltdb v0.0.0-20220329195025-15018e9b97e0
 	github.com/ydb-platform/ydb-go-sdk-auth-environ v0.1.2
 	github.com/ydb-platform/ydb-go-sdk/v3 v3.25.3
+	go.uber.org/atomic v1.9.0
```

Comment:
can you use https://stackoverflow.com/a/22035083/997701 instead of another library?